javalib/src/main/scala/java/lang/Long.scala

```diff
@@ -36,82 +48,339 @@ object Long {
   final val SIZE      = 64
   final val BYTES     = 8
 
-  @inline def toString(i: scala.Long, radix: Int): String = ???
+  @inline def bitCount(l: scala.Long): scala.Int =
+    Intrinsics.`llvm.ctpop.i64`(l).toInt
 
-  @inline def toUnsignedString(i: scala.Long, radix: Int): String = ???
+  @inline def compare(x: scala.Long, y: scala.Long): scala.Int =
+    if (x == y) 0
+    else if (x < y) -1
+    else 1
 
-  @inline def toString(i: scala.Long): String = ???
+  @inline def compareUnsigned(x: scala.Long, y: scala.Long): scala.Int =
+    compare(x ^ scala.Long.MinValue, y ^ scala.Long.MinValue)
+
+  def decode(nm: String): Long = {
+    val length = nm.length()
+    if (length == 0) {
+      throw new NumberFormatException()
+    } else {
+      var i          = 0
+      var firstDigit = nm.charAt(i)
+      val negative = firstDigit == '-'
+
+      if (negative) {
+        if (length == 1) {
+          throw new NumberFormatException(nm)
+        }
+
+        i += 1
+        firstDigit = nm.charAt(i)
+      }
+
+      var base = 10
+      if (firstDigit == '0') {
+        i += 1
+        if (i == length) {
+          return valueOf(0L)
+        }
+
+        firstDigit = nm.charAt(i)
+        if (firstDigit == 'x' || firstDigit == 'X') {
+          if (i == length) {
+            throw new NumberFormatException(nm)
+          }
+          i += 1
+          base = 16
+        } else {
+          base = 8
+        }
+      } else if (firstDigit == '#') {
+        if (i == length) {
+          throw new NumberFormatException(nm)
+        }
+        i += 1
+        base = 16
+      }
+
+      valueOf(parse(nm, i, base, negative))
+    }
+  }
 
-  @inline def toUnsignedString(i: scala.Long): String = ???
+  @inline
+  def divideUnsigned(dividend: scala.Long, divisor: scala.Long): scala.Long =
+    divULong(dividend, divisor)
 
-  def parseLong(s: String, radix: Int): scala.Long = ???
+  def getLong(nm: String): Long =
+    if (nm == null || nm.length() == 0) {
+      null
+    } else {
+      val prop = System.getProperty(nm)
+      if (prop == null) {
+        null
+      } else {
+        try {
+          decode(prop)
+        } catch {
+          case e: NumberFormatException =>
+            null
+        }
+      }
+    }
+
+  def getLong(nm: String, v: scala.Long): Long =
+    if (nm == null || nm.length() == 0) {
+      valueOf(v)
+    } else {
+      val prop = System.getProperty(nm)
+      if (prop == null) {
+        valueOf(v)
+      } else {
+        try {
+          decode(prop)
+        } catch {
+          case e: NumberFormatException =>
+            valueOf(v)
+        }
+      }
+    }
+
+  @inline def getLong(nm: String, v: Long): Long =
+    getLong(nm, v.longValue)
```

Comment:
Fixed in https://github.com/scala-native/scala-native/pull/238/commits/f0eab14d727ad8978b63a07733f4f4a89798198f